linux/guest_customization/gosc_support_matrix.yml

```diff
@@ -101,3 +101,16 @@ OracleLinux:
       7.0.1: N/A
       7.0.2: N/A
       7.0.3: 20051473
+
+# Pardus 21.x / 23.x support GOSC testing on vCenter Server 8.0U2 and on vCenter Server 7.0.3 from build 22585855(7.0.3P08), or higher vCenter Server versions.
+Pardus GNU/Linux:
+  '21.0':
```

Comment:
This project supports Pardus since 21.2, so it is better to change the start version to 21.2